.vscode/tasks.json

```diff
@@ -0,0 +1,41 @@
+{
+	"version": "2.0.0",
+	"tasks": [
+    {
+      "type": "docker-build",
+      "label": "docker-build",
+      "platform": "python",
+      "dockerBuild": {
+        "tag": "opentransitmetrics:flask-dev",
+        "dockerfile": "${workspaceFolder}/Dockerfile",
+        "context": "${workspaceFolder}",
+        "pull": true,
+        "target": "flask-dev",
+        "customOptions": "${input:dockerBuildm1}",
+      }
+    },
+    {
+      "type": "docker-run",
+      "label": "docker-run: save_routes.py",
+      "dependsOn": [
+        "docker-build"
+      ],
+      "python": {
+        "args": ["--s3", "--timetables", "--scheduled-stats", "--agency=trimet"],
```

Comment:
I think it might be worthwhile to shift this towards a user-specified command rather than this hardcoded option opportunistically as other tickets are worked on that involve debugging other scripts. I'm open to giving that a go now though if that seems better!

Reply:
Let's keep this format for now. I also like the idea of a user-specified command but I think that can be a v2.